index.html

```diff
@@ -46,6 +46,9 @@
     </script>
     -->
 
+    <!-- Add Proj4js -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/proj4js/2.5.0/proj4.js"></script>
```

Comment:
I'd prefer to include proj4.js in the `app.json` as it is done with OpenLayers. The `app.json` (or better its "js" section) also accepts remote files.

Reply:
Alright, moved dependency to `app.json` in 4db9297